lib/MusicBrainz/Server/Controller/Collection.pm

```diff
@@ -296,6 +297,33 @@ sub edit : Chained('own_collection') RequireAuth {
     );
 }
 
+with 'MusicBrainz::Server::Controller::Role::Merge';
+
+sub _merge_form_arguments {
+    my ($self, $c, @collections) = @_;
+
+    return (requires_edit_note => 0);
+}
+
+sub _merge_load_entities {
+    my ($self, $c, @collections) = @_;
+    $c->model('Collection')->load_entity_count(@collections);
+    $c->model('CollectionType')->load(@collections);
+
+    for my $collection (@collections) {
+        $c->model('Editor')->load_for_collection($collection);
+    };
+
+    my @entity_types = uniq map { $_->type->item_entity_type } @collections;
+    my @privacy_settings = uniq map { $_->public } @collections;
+    use Data::Dumper;
+    print Dumper(\@privacy_settings);
+    $c->stash(
+        privacy_differs => @privacy_settings > 1,
```

Comment:
"privacies_differ" would be a bit more consistent with types_differ

Reply:
It just sounded kinda stupid :D But I guess.